packages/create-svelte/shared/+vitest/vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+
+/** @type {import('vite').UserConfig} */
+const config = {
+	plugins: [sveltekit()],
+	test: {
+		include: ['src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
```

Comment:
should we reduce the pattern to less combinations? projects are always esm so we could just use one extension (.js for js projects, .ts for typescript).

If and do we need both spec and test postfixes?

This is not a request, more the start of a mini-discussion

Reply:
agree with that latter suggestion — updated